tools/lint/lint.py

```diff
@@ -898,7 +898,9 @@ def create_parser():
                         help="The WPT directory. Use this "
                         "option if the lint script exists outside the repository")
     parser.add_argument("--ignore-glob", type=ensure_text, action="append",
-                        help="Additional file glob to ignore (repeat to add more)")
+                        help="Additional file glob to ignore (repeat to add more). "
+                        "Globs are matched against full paths using fnmatch; "
+                        "separators are not treated specially.")
```

Comment:
It's not clear to me what 'separators are not treated specially' means. Is this referring to us not normalizing separators? (Which may be a problem, if someone ever does want to ignore `foo/bar*` ?)

Reply:
I know a few:
1. Path separators are not normalized, as you said
2. "*" can match across path separators (e.g. `*foo` matches `bar/foo`)

There might be more if I keep reading the implementation. The language comes from stdlib docs: https://docs.python.org/3/library/fnmatch.html#index-3:~:text=Note%20that%20the%20filename%20separator%20('%2F'%20on%20Unix)%20is%20not%20special%20to%20this%20module.

Reply:
Oh shoot. We actually have a wrapper for `fnmatch` to normalize path separators... https://github.com/web-platform-tests/wpt/blob/master/tools/lint/fnmatch.py

But point 2 still stands.

Reply:
Hrm, so something like: "Globs are matched against full paths using fnmatch, except that path separators are normalized." is probably better? The idea being you would look up the fnmatch documentation if you wanted to know the behavior.

Reply:
Updated to
                        "Globs are matched against paths relative to REPO_ROOT "
                        "using fnmatch, except that path separators are normalized."